src/main/java/com/minecolonies/coremod/colony/requestsystem/management/IStandardRequestManager.java

```diff
@@ -24,10 +25,10 @@
     BiMap<IToken<?>, IRequestResolverProvider> getProviderBiMap();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)